endpoints/openrtb2/auction.go

```diff
@@ -1068,6 +1185,8 @@ func isBidderToValidate(bidder string) bool {
 		return false
 	case openrtb_ext.BidderReservedSKAdN:
 		return false
+	case openrtb_ext.BidderReservedBidder:
+		return false
```

Comment:
Nitpick: Can we add a unit test for the added `case` statement?
![image](https://user-images.githubusercontent.com/15863840/137958876-bfd221e5-a837-4e2c-be38-c1492b8b5500.png)